dashboard/app/models/course.rb

```diff
@@ -362,6 +362,30 @@ def select_course_script(user, default_course_script)
     default_course_script
   end
 
+  # If the user does not have progress on any script in the current course, but
+  # has progress on a script in another course in the same course family, then
+  # they are on the wrong version of the course.
```

Comment:
Doesn't this mean they are _probably_ in the wrong version? Might be worth noting in comments.